PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -129,6 +140,10 @@ fi
 #display the parsed/default values
 opts_ShowValues
 
+if [ "$LongitudinalMode" == "NONE" && "$Species" != "Human" ]; then 
```

Comment:
`"$LongitudinalMode" == "NONE"` is probably the wrong condition to test here, currently this appears to trip when nonhuman is run non-longitudinally.